api/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.openmrs.module</groupId>
 		<artifactId>hospitalcore</artifactId>
-		<version>1.0.7-SNAPSHOT</version>
+		<version>1.0.8-SNAPSHOT</version>
```

Comment:
This the current version https://github.com/uonafya/ehrhospitalcoreopenmrs/blob/openmrs2.3.3/pom.xml#L8 in 2.x, any reason why you defaulted to 1.x